### DISCUSSION TITLE
How to run code on my local machine(frontend)

### DISCUSSION THREAD

**Discussion by rajanshresth** [ToolJet/ToolJet#5452] [General]
I've cloned the project on my local machine.

**Comment by withshubh** [comment]
Please follow the contributing guides here: https://docs.tooljet.com/docs/contributing-guide/setup/macos

**Reply by rajanshresth** [reply]
do i need specific version of node also(like npm version 7.20.0)

**Comment by rajanshresth** [comment]
I've followed the contributing guides.
It shows the error,
"sh: 1: webpack: not found"

**Reply by rajanshresth** [reply]
I've followed the contributing guides.
It shows the error,
"sh: 1: webpack: not found"

**Reply by rajanshresth** [reply]
do i need specific version of node also(like npm version 7.20.0)

**Reply by withshubh** [reply]
Yes, it requires:
- Node version 14.17.3
- npm version 7.20.0
